cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java

```diff
@@ -60,19 +63,29 @@
  */
 public final class TokenSecureStoreUpdater implements SecureStoreUpdater {
   private static final Logger LOG = LoggerFactory.getLogger(TokenSecureStoreUpdater.class);
+
   private final YarnConfiguration hConf;
   private final LocationFactory locationFactory;
+  private final SecureStore secureStore;
   private final long updateInterval;
   private final boolean secureExplore;
+  private final boolean isKMSBacked;
 
   @Inject
-  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf, LocationFactory locationFactory) {
+  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf,
+                                 LocationFactory locationFactory, SecureStore secureStore) {
     this.hConf = hConf;
     this.locationFactory = locationFactory;
+    this.secureStore = secureStore;
     secureExplore = cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED) && UserGroupInformation.isSecurityEnabled();
+    isKMSBacked = isKMSBackedStore(cConf, secureStore);
```

Comment:
also, we follow the convention that members should not begin with ``is`` ( so you can have methods/getters that return them to be ``is<memberName>``